packages/rke2-canal/charts/values.yaml

```
@@ -60,19 +60,19 @@ calico:
   # CNI installation image.
   cniImage:
     repository: rancher/hardened-calico
-    tag: v3.29.0-build20241104
+    tag: v3.29.1-build20241126
```

```suggestion
    tag: v3.29.1-build20241127
```

The publish-build action failed for `-build20241126`:
https://github.com/rancher/image-build-calico/actions/runs/12023382802/job/33517164775

Resolved by:
https://github.com/rancher/ecm-distro-tools/pull/506

with a new build suffix `-build20241127`